### PR TITLE
Improve spec goal. See #26

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

Integrate @darrelmiller suggestions in the spec goal

- [x]  Simplify API documentation by eliminating the need to include detailed quota limits and related header fields in API documentation.

### CODE REVIEW

**Review by unleashed — COMMENTED** [review]
This looks ok, but let's decide first what to do about the second item. Perhaps it can be left focused around service and infrastructure limits and this third one can be reworded around easing adoption via factors such as simplified API docs or compatibility with existing solutions.

**Comment by ioggstream** [comment]
> let's decide first what to do about the second item

agree: let's discuss that in #26 then.

**Review by unleashed — APPROVED** [review]
LGTM

**Comment by ioggstream** [comment]
thx @unleashed . Since the issue was open by @darrelmiller I'll wait for his review

**Comment by ioggstream** [comment]
@darrelmiller PTAL :)

File: draft-ietf-httpapi-ratelimit-headers.md [hunk 1]
```diff
@@ -145,13 +145,17 @@ using multiple and variable time windows and dynamic quotas, or implementing con
 
 The goals of this proposal are:
 
-   1. Standardizing the names and semantic of rate-limit headers;
+   1. Standardizing the names and semantic of rate-limit headers
```
Comment (darrelmiller) [inline draft-ietf-httpapi-ratelimit-headers.md:148]:
```suggestion
   1. Standardizing the names and semantics of rate-limit headers
```

File: draft-ietf-httpapi-ratelimit-headers.md [hunk 2]
```diff
@@ -145,13 +145,17 @@ using multiple and variable time windows and dynamic quotas, or implementing con
...
+      to ease their enforcement and adoption;
 
-   2. Improve resiliency of HTTP infrastructures simplifying
-      the enforcement and the adoption of rate-limit headers;
+   2. Improve resiliency of HTTP infrastructures
```
Comment (darrelmiller) [inline draft-ietf-httpapi-ratelimit-headers.md:151]:
```suggestion
   2. Improve resiliency of HTTP infrastructure by
```

**Comment by darrelmiller** [comment]
Looks good. I just made some minor language suggestions.

**Comment by ioggstream** [comment]
> Looks good. I just made some minor language suggestions.

Ok, merging thanks!

**Pull request merged**